### PR TITLE
tests: Add a couple of line continuation tests.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Add a couple of tests for line continuation corner cases.

Issue #1269

### CODE REVIEW

File: tests/test_edgeql_expressions.py [hunk 1]
```diff
@@ -3085,6 +3085,32 @@ class TestExpressions(tb.QueryTestCase):
...
+        # Issue #1269
+        #
+        # \r or \n should both be allowed after line continuation symbol
+        await self.assert_query_result(
+            "SELECT 'bb\\\n   aa';",
+            ['bbaa'],
+        )
+
+        await self.assert_query_result(
+            "SELECT 'bb\\\r   aa';",
```
Comment (tailhook) [inline tests/test_edgeql_expressions.py]: This should probably has `\r\n` or another test for `\r\n` because turning `'a\\\r\nb` into `a\nb` sounds bad.

File: tests/test_edgeql_expressions.py [hunk 2]
```diff
@@ -3057,7 +3057,7 @@ class TestExpressions(tb.QueryTestCase):
 
             bb';
             ''',
-            ['bbaa bb'],
+            ['bbaa \n\n            bb'],
```
Comment (vpetrovykh) [inline tests/test_edgeql_expressions.py]: Actually, I strongly disagree with this change. The whole point of our `\` is that is consumes all white-space until next non-white-space, thus allowing to format a single logical string as you like. Consider this:
```
SELECT 'Always remember that: "\

        Long strings are very \
        useful because they let \
        the client make rendering \
        decisions.\

        " is all in one line (or, perhaps, \
        one paragraph)';
```

The point is that we wanted to find a way to make our multi-line strings somewhat easier to format as single logical line and only folding a single line after `\` is not that helpful. In practice I would never use that feature if I had several newlines, but as my example above shows I would be quite happy with the option of visually separating text blobs with newlines while keeping them all a contiguous string.

@1st1, @elprans what do you think about handling whitespace after `\`?

Comment (1st1) [reply tests/test_edgeql_expressions.py]: I also disagree with the change. Let's handle `\` the way rust handles them:

Screen Shot 2020-03-05 at 12 17 27 PM

Comment (1st1) [reply tests/test_edgeql_expressions.py]: That means that the behavior we implemented in a2 and current master branch is the correct one.

Comment (tailhook) [reply tests/test_edgeql_expressions.py]: Okay. Fixed that
